uaaextras/clients/cf.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from cloudfoundry_client.client import CloudFoundryClient
+from cloudfoundry_client.errors import InvalidStatusCode
+
+logger = logging.getLogger(__name__)
+
+def get_cf_client(target_endpoint, access_token):
+    client = CloudFoundryClient(target_endpoint)
+    client._access_token = access_token
+    return client
+
+def is_org_manager(client, user_id):
+    if 'organization_manager' in client.v3.roles.list(user_guids=user_id):
+        return True
+    else:
+        return False
```

Comment:
maybe just:
```
return 'organization_manager' in client.v3.roles.list(user_guids=user_id)
```

Reply:
Changed to properly grab organization manager